tests/system/fim/common_tasks/verify_alerts_elasticsearch.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+
+import argparse
+from elasticsearch import Elasticsearch
+
+
+def setElasticsearch(ElasticIP):
+    """
+        Sets the Elasticsearch instance that we want to connect.
+        :param str ElasticIP: IP adress of the Elasticsearch node.
+        :return: Object of the Elasticsearch class.
+    """
+    es = Elasticsearch("http://{}:9200".format(ElasticIP))
+    return es
+
+
+def makeQuery(query, Elastic, index_name):
+    """
+        Make a query to a Elasticsearch instance.
+        :param Dict query: Dictionary containing the query we want to make,
+        must follow Query DSL.
+        :param Elasticsearch Elastic: Elasticsearch class instance we want
+        to query.
+        :param str index_name: Index  of the Elasticsearch instance we want
+        to query.
+        :return: Dictionary containing the result of the query.
+    """
+    result = Elastic.search(index=index_name, body=query)
+    return result
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-f", "--files", type=str, required=True, dest='files',
+        help="File containing the list of modified files, one per line"
+    )
+    parser.add_argument(
+        "-e", "--event", type=str, required=True, dest='alert',
+        choices=['added', 'modified', 'deleted'],
+        help="Type of event that we expect: added, modified, deleted"
+    )
+    parser.add_argument(
+        "-i", "--ip", type=str, required=True, dest='ip',
```

Comment:
This should be adapted after adopting a convention of https://github.com/wazuh/wazuh-qa/issues/483#issuecomment-587059411